### PR TITLE
Updates in a workflow

### PR DESCRIPTION
**Pull request by danimartin-dev**

### CODE REVIEW

**Comment by dannymartinm** [comment]
test

**Review by dannymartinm — COMMENTED** [review]
test

File: .github/workflows/seed-job-new-file.yml [hunk 1]
```diff
@@ -17,4 +17,5 @@ jobs:
       uses: actions/checkout@v2
       with:
         ref: ${{ inputs.SHA }}
-    - run: echo 'Hello from test ///////////////// ' $GITHUB_SHA ${{ inputs.SHA }}
+    - run: echo 'Hello from PR in danimartin-dev ///////////////// '
+    - run: echo $GITHUB_SHA ${{ inputs.SHA }}
```
Comment (dannymartinm) [inline .github/workflows/seed-job-new-file.yml]: test 2

**Review by dannymartinm — APPROVED** [review]
test 3

**Comment by dannymartinm** [comment]
test

**Review by dannymartinm — APPROVED** [review]
approving cmmnt and action

**Comment by dannymartinm** [comment]
PR  comment

**Comment by danimartin-dev** [comment]
PR COmment

**Review by danimartin-dev — COMMENTED** [review]
Diff Comment

**Review by dannymartinm — COMMENTED** [review]
diff comment

**Review by dannymartinm — APPROVED** [review]
diff comment and approve

**Comment by dannymartinm** [comment]
new comment

**Comment by dannymartinm** [comment]
test cmmnt

**Review by dannymartinm — COMMENTED** [review]
new comment

**Review by dannymartinm — APPROVED** [review]
new comment approved

**Comment by dannymartinm** [comment]
cmmnt

**Review by dannymartinm — COMMENTED** [review]
diff cmnnt

**Review by dannymartinm — APPROVED** [review]
deedede

**Comment by dannymartinm** [comment]
single comment

**Review by dannymartinm — COMMENTED** [review]
commnt in diff

**Comment by dannymartinm** [comment]
test

**Review by dannymartinm — APPROVED** [review]
cmmt + app

**Review by dannymartinm — APPROVED** [review]
app+cmnt

**Review by dannymartinm — COMMENTED** [review]
single comment in. diff

**Review by dannymartinm — COMMENTED** [review]
sc sr

**Comment by dannymartinm** [comment]
cccc

**Comment by dannymartinm** [comment]
cmmnr

**Comment by dannymartinm** [comment]
test

**Review by dannymartinm — APPROVED** [review]
tst diff

**Comment by dannymartinm** [comment]
test

**Review by dannymartinm — APPROVED** [review]
aaaa

**Comment by dannymartinm** [comment]
test

**Review by dannymartinm — COMMENTED** [review]
test

**Comment by dannymartinm** [comment]
123

**Comment by dannymartinm** [comment]
rrr

**Review by dannymartinm — COMMENTED** [review]
test

**Review by dannymartinm — COMMENTED** [review]
testing

**Review by dannymartinm — COMMENTED** [review]
test

**Review by dannymartinm — COMMENTED** [review]
review

**Review by dannymartinm — COMMENTED** [review]
test

**Review by dannymartinm — APPROVED** [review]
ctest

**Comment by dannymartinm** [comment]
test

**Comment by dannymartinm** [comment]
test

**Comment by dannymartinm** [comment]
test

**Review by dannymartinm — COMMENTED** [review]
lgtm

**Comment by dannymartinm** [comment]
test

**Review by dannymartinm — COMMENTED** [review]
test

**Review by dannymartinm — APPROVED** [review]
LGTM!

**Comment by dannymartinm** [comment]
test2

**Review by danimartin-dev — COMMENTED** [review]
test fom danimartin-dev

**Review by dannymartinm — COMMENTED** [review]
seed job

**Review by dannymartinm — APPROVED** [review]
Seed Job

**Review by dannymartinm — APPROVED** [review]
gf

**Review by dannymartinm — APPROVED** [review]
LGTM!

**Review by dannymartinm — APPROVED** [review]
lgtm. thanks

**Review by dannymartinm — APPROVED** [review]
run seed job

**Pull request closed without merging**